cmd/podman/shared/create.go

```diff
@@ -77,7 +77,11 @@ func CreateContainer(ctx context.Context, c *GenericCLIResults, runtime *libpod.
 			writer = os.Stderr
 		}
 
-		newImage, err := runtime.ImageRuntime().New(ctx, c.InputArgs[0], rtc.SignaturePolicyPath, GetAuthFile(""), writer, nil, image.SigningOptions{}, false, nil)
+		name := ""
+		if len(c.InputArgs) != 0 {
```

Comment:
It seems like it ought to be an error if this isn't true - we can give a much clearer error message than what pulling an empty-string image would give.